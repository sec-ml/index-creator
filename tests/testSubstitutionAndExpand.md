| term             | sub-term                           | notes                   | book | page             |
| ---------------- | ---------------------------------- | ----------------------- | ---- | ---------------- |
| Linux built-in<> | grep && cat && tail && cd && mkdir | linux built-in commands | 1    | **10**, 1, 15-16 |
