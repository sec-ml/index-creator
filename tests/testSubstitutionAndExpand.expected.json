[
  {
    "term": "cat",
    "subTerm": "Linux built-in",
    "notes": "linux built-in commands",
    "book": 1,
    "page": "1, **10**, 15-16"
  },
  {
    "term": "cd",
    "subTerm": "Linux built-in",
    "notes": "linux built-in commands",
    "book": 1,
    "page": "1, **10**, 15-16"
  },
  {
    "term": "grep",
    "subTerm": "Linux built-in",
    "notes": "linux built-in commands",
    "book": 1,
    "page": "1, **10**, 15-16"
  },
  {
    "term": "Linux built-in",
    "subTerm": "cat",
    "notes": "linux built-in commands",
    "book": 1,
    "page": "1, **10**, 15-16"
  },
  {
    "term": "Linux built-in",
    "subTerm": "cd",
    "notes": "linux built-in commands",
    "book": 1,
    "page": "1, **10**, 15-16"
  },
  {
    "term": "Linux built-in",
    "subTerm": "grep",
    "notes": "linux built-in commands",
    "book": 1,
    "page": "1, **10**, 15-16"
  },
  {
    "term": "Linux built-in",
    "subTerm": "mkdir",
    "notes": "linux built-in commands",
    "book": 1,
    "page": "1, **10**, 15-16"
  },
  {
    "term": "Linux built-in",
    "subTerm": "tail",
    "notes": "linux built-in commands",
    "book": 1,
    "page": "1, **10**, 15-16"
  },
  {
    "term": "mkdir",
    "subTerm": "Linux built-in",
    "notes": "linux built-in commands",
    "book": 1,
    "page": "1, **10**, 15-16"
  },
  {
    "term": "tail",
    "subTerm": "Linux built-in",
    "notes": "linux built-in commands",
    "book": 1,
    "page": "1, **10**, 15-16"
  }
]
